### PR TITLE
Configurations Validator Tool (cerberus)

### PR DESCRIPTION
**Pull request by SYNchroACK**

### CODE REVIEW

**Comment by wagner-certat** [comment]
I like the idea.

Do you know `intelmqctl check`? Does the same thing basically. I'd like ti have only one tool to check things, not a bunch of them (and as actual executable in the installation).

**Comment by SYNchroACK** [comment]
What do you suggest @wagner-certat ? Merge this validator in intelmqctl `def check`?

File: intelmq/bin/configs-validators/intelmq-conf-validator [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import sys
+
+try:
+    import cerberus
+except:
+    print("[-] Please install cerberus using the following command: 'pip install cerberus'.", file=sys.stderr)
```
Comment (wagner-certat) [inline intelmq/bin/configs-validators/intelmq-conf-validator]: The package is called `Cerberus` with a capital `C`.

File: intelmq/bin/configs-validators/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  --config-file <filepath>
+                        config file (JSON or YAML)
+  --validation-file <filepath>
+                        validation file
+```
+
+```
+cd intelmq/intelmq/bin/validators
+
+./intelmq-conf-validator --config-file=../../etc/feeds.yaml --validation-file=feeds.schema.json
```
Comment (wagner-certat) [inline intelmq/bin/configs-validators/README.md]: This file is not in that branch ;)

**Comment by wagner-certat** [comment]
> Contrary to the documentation YAML is not supported:

Update: Both the schema was incomplete and the file was invalid

**Comment by wagner-certat** [comment]
I now moved the code to `tests/test_conf`

**Pull request merged**